### DISCUSSION TITLE
打不开几天前的跟读的音频了

### DISCUSSION THREAD

**Discussion by SkyPan19** [ZuodaoTech/everyone-can-use-english#714] [Q&A]
【MEDIA_ELEMENT_ERROR: Format error
解码波形失败: enjoy://library/audios/0c36f112a27d55bb0e4940a3e95cf8cc.mp3】
![Uploading 屏幕截图 2024-06-25 160927.png…]()

**Comment by an-lee** [comment]
是不是动过 `/EnjoyLibrary` 文件夹里面的文件？

**Reply by SkyPan19** [reply]
right！

**Reply by an-lee** [reply]
为啥要动这里面的文件？想要源文件在 Enjoy 里可以直接下载。

源文件改动了，Enjoy 自然就找不到了，也就无法播放了。除非能复原，否则就把这条音频删了吧。

**Comment by SkyPan19** [comment]
你好，跟读时，设置了单句循环模式，选中跟读一句话中的后半段时，每次系统播放循环的间隔时间有点长，有好几秒，这影响跟读进度。等不及它播放时，当我再次按空格播放时又会和系统的自动播放冲突；所以单句循环模式的间隔时间能缩短吗？
----- 原始邮件 -----
发件人：an-lee ***@***.***>
收件人：xiaolai/everyone-can-use-english ***@***.***>
抄送人：SkyPan19 ***@***.***>,  Author ***@***.***>
主题：Re:_[xiaolai/everyone-can-use-english]_打不开几天前的跟读的音频了_(Discussion_#714)
日期：2024年06月25日 20点33分


为啥要动这里面的文件？想要源文件在 Enjoy 里可以直接下载。
源文件改动了，Enjoy 自然就找不到了，也就无法播放了。除非能复原，否则就把这条音频删了吧。
